### ISSUE TITLE
Update documentation

### ISSUE THREAD

**Issue by kaikreuzer** [openhab/openhab-qnap-qpkg#17]
Hi @thopiekar,

many thanks for putting together a [package with the final 2.0 release](https://github.com/openhab/openhab-qnap-qpkg/releases/tag/2.0.0-0)!

I'd like to mention this on social media, so that people know about it. I wonder if you could update the screenshots to reflect that release?
[Here in the README.md](https://github.com/openhab/openhab-qnap-qpkg/blob/master/README.md) and on http://docs.openhab.org/installation/qnap.html it shows version 1.9.9.b3 and a file openHAB_2.0.0-b3-1.qpkg in the images. This might be irritating for a few users, so an update would be great!

**Comment by thopiekar** [comment]
Done. Here in my repo..
Only needs an update in openhab.docs..

**Comment by 6r15u** [comment]
@kaikreuzer how can the double work with the readme.md here and the documentation in openhab-docs be reduced? I checked and makeup langugae doesn't allow include of other pages.
Is referring of wiki (openhab-docs) to the readme here or from here to the wiki better?

**Comment by kaikreuzer** [comment]
@6r15u We usually maintain the content in the repo itself (either as the README.md or as an additional USAGE.md) and then have it copied into the openhab-docs like e.g. here:
https://github.com/openhab/openhab-docs/blob/5c7229bfc07969e2d0e3d243e5594ca73f91ee3b/pom.xml#L463-L473

@ThomDietrich Might be able to help you on finding the best way here.

**Comment by ThomDietrich** [comment]
@6r15u exactly. It would be nice to do just that with this repository as well. All you need to do is provide a USAGE.md file. Please look at these similar issues/prs where I've already described the process and things to do:

* https://github.com/openhab/openhab-android/issues/495
* https://github.com/openhab/openhab-docs/pull/594
* https://github.com/openhab/openhab-android/pull/509

**Comment by omachala** [comment]
Hi! I can help update these documentation screenshots.

I maintain [heroshot](https://heroshot.sh), a screenshot automation tool that captures screenshots from a config file using Playwright. Once set up, regenerating screenshots becomes a one-command operation.

Happy to help if this is still needed!
